.circleci/config.yml

```diff
@@ -35,23 +35,24 @@ install_official_git_client: &install_official_git_client
   command: |
     set -ex
 
-    sudo systemctl stop apt-daily.service
-    sudo systemctl kill --kill-who=all apt-daily.service
+    sudo systemctl stop apt-daily.service || true
```

Comment:
is it worth trying to do?:
 apt-get uninstall -y unattended-upgrades apt-listchanges

Reply:
It's possible. But I also don't see any reason to prefer that over disabling the systemd service?

Reply:
if the software is installed, I wouldn't be 100% sure that it is not running (could be a bug or something else that enabled it without us knowing)

removing the code does make me feel 100% sure it is not running

and also, if there is a funny thing going on that reinstalls it without us knowing, we will know

Reply:
there could be chef or other automatic security/remote management tools that decide to reinstall it and enable it

Reply:
We cannot uninstall `apt-daily.service`, it comes with `apt` package and we cannot uninstall that for obvious reasons.

Reply:
I see, but at least we can uninstall unattended-upgrades

Reply:
`apt purge unattended-upgrades` has a chicken-egg problem. If apt/dpkg is running because of unattended-upgrades, you cannot remove it (because you don't have the lock). So you still need to terminate the service first and wait for it to exit, before you can remove it.